core/unix/signal.c

```diff
@@ -3423,6 +3423,7 @@ transfer_from_sig_handler_to_fcache_return(dcontext_t *dcontext, kernel_ucontext
      * from DR's handler.
      */
     ASSERT(get_sigcxt_stolen_reg(sc) != (reg_t)*get_dr_tls_base_addr());
+    // XXX i#4495: Consider saving stolen reg's application value.
```

Comment:
I would again differentiate the *translated* value.

Reply:
Done.